dpctl/_sycl_event.pyx

```diff
@@ -202,3 +206,18 @@ cdef class SyclEventRaw(_SyclEventRaw):
             "SyclEventRef",
             &_event_capsule_deleter
         )
+
+    def execution_status(self):
```

Comment:
Perhaps this should be a read-only property. 

```python
In [2]: dpctl.SyclEventRaw().execution_status
Out[2]: <function SyclEventRaw.execution_status>

In [3]: dpctl.SyclEventRaw().execution_status()
Out[3]: <event_status_type.complete: 4>
```

So one would use `In[2]` rather than `In[3]`.

Reply:
```suggestion
    @property
    def execution_status(self):
```